docs-web/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -1024,7 +1024,8 @@ codes: [401, 403]
 return:
   code: 200
   type: application/json
-  body: "{\"msg\": \"You don't have permission to do this\"}"
+  body: |
+    {\"msg\": \"You don't have permission to do this\"}
```

Comment:
because the string itself is not multi-line, why not use:
```yaml
body: '{\"msg\": \"You do not have permission to do this\"}'
```

Reply:
It's defined like that in the Ticket. Also mixing `"` with `'` looks weird to me.

Reply:
ok. thanks